docs/docs/user-guide/permissions.md

```diff
@@ -0,0 +1,108 @@
+- **Define Child Roles**: PostgreSQL has a mechanism for [Role Membership](https://www.postgresql.org/docs/current/role-membership.html) wherein any role can be "granted" to any other role to form simple hierarchies or complex graph-based inheritance structures. For any role you've configured within Mathesar, you can use Mathesar to grant the role to other "child roles".
+- **Drop Roles**: You can drop server-level roles that are no longer needed. This action removes the role from the server and all databases where it has been assigned. Exercise caution when dropping roles, as it may affect existing permissions and user access across multiple databases.
+
+!!! note
+    Server roles, once added, must be configured in Mathesar under the **Role Configuration** section before they can be assigned to collaborators.
+
+---
+
+## Permissions Management at Different Levels
+
+Mathesar manages permissions at three levels:
+
+1. **Database Level:** Managed from the database page.
+2. **Schema Level:** Managed from the schema page.
+3. **Table Level:** Managed from the inspector panel for each table.
+
+### Ownership of Objects
+
+Owners of each object (database, schema, or table) can set permissions for other users. Ownership is determined by the user's assigned collaborator role in the database.
+
+When a new object (database, schema, or table) is created, the owner of the object is the collaborator who created it. This automatic ownership assignment allows the creator to have immediate and full control over the objects they create, including the ability to manage permissions and transfer ownership if needed.
+
+### Database Permissions
+
+Database permissions control access and actions at the database level.
+
+- **Owner**: Each database has an owner who has full control over the database, including managing permissions and transferring ownership.
+- **Granted Access**: Specific permissions can be granted to roles for various actions within the database.
+- **Transfer Ownership**: The current owner can transfer ownership to another role, granting them full administrative control.
+
+For each database, the following permission levels can be granted:
+
+- **Connect**: Allows the role to access and connect to the database.
+- **Create**: Includes Connect permissions and allows the role to create new schemas within the database.
+- **Custom**: Enables the granular setting of permissions beyond the predefined options.
+
+### Schema Permissions
+
+Schema permissions control access and actions at the schema level.
+
+- **Owner**: Each schema has an owner who has full control over the schema, including managing permissions and transferring ownership.
+- **Granted Access**: Specific permissions can be granted to roles for various actions within the schema.
+- **Transfer Ownership**: The current owner can transfer ownership to another role, granting them full administrative control over the schema.
```

Comment:
> full administrative control

Can we remove the word `full`?

`administrative control` sounds great!

Reply:
I've removed the word "full" from all instantances.